web/src/compose_notifications.ts

```diff
@@ -91,6 +92,7 @@ export function notify_automatic_new_visibility_policy(
             classname: compose_banner.CLASSNAMES.automatic_new_visibility_policy,
             link_msg_id: data.id,
             channel_name: message_recipient.channel_name,
+            topic_name: message_recipient.topic_name,
```

Comment:
Ideally, we should use a `should_include_topic_name_attr` here and at all the places where `compose_banner` is used. But that'd create a kind of churn and we'll still need to make sure that `topic_name` is passed if `should_include_topic_name_attr` is true.

Simply passing `topic_name` is a simple solution here, even though it is unused for this banner.

Let me know if we should do the first version or any better suggestion you have.

Reply:
This seems OK, but let's leave this as a comment.

Reply:
Wrote a simple comment for this. Eventually, we'll have a typed system replace handlebars to make this sort of bug impossible.